chall03/tdawson.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+func execTime(start time.Time) {
+	fmt.Printf("Finished in %dms\n", time.Since(start).Milliseconds())
+}
+
+func httpGet(url string) string {
```

Comment:
I would not suggest create a function httpGet, but use `http.Get()` inside your main scope. So if later you want to improve and use response Header or status code in your main code it's easier and cleaner.